misc/docker/centos/7/Dockerfile

```diff
@@ -1,6 +1,20 @@
 FROM centos:7
+ARG test
 RUN yum install -y git gcc make epel-release
 RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY*
+
+# update gcc  version
```

Comment:
default version of gcc in centos 7 is 4.8.5, so I updated gcc version for `ASAN=1 unittest`